packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -670,6 +699,10 @@ function useRestoreFocus(scopeRef: RefObject<Element[]>, restoreFocus: boolean,
  */
 export function getFocusableTreeWalker(root: Element, opts?: FocusManagerOptions, scope?: Element[]) {
   let selector = opts?.tabbable ? TABBABLE_ELEMENT_SELECTOR : FOCUSABLE_ELEMENT_SELECTOR;
+  if (root === null) {
```

Comment:
This technically fixes the exception, but it just moves the exception elsewhere which is why the code above exists to change scope.